node/src/effect.rs

```diff
@@ -815,17 +815,21 @@ impl<REv> EffectBuilder<REv> {
     pub(crate) async fn request_proto_block(
         self,
         block_context: BlockContext,
+        next_block_height: u64,
```

Comment:
Can we remove the TODO on this method? :thinking:

Reply:
I'd be for it. It doesn't seem relevant, and if we need to change the types at some point, we'll know it, anyway.